pkg/machines/components/vmDisksTab.jsx

```diff
@@ -80,13 +79,7 @@ class VmDisksTab extends React.Component {
     }
 
     open() {
-        // Refresh storage volume list before displaying the dialog.
```

Comment:
Hmm I am not sure I understand why you do this change.
You need to refresh the pools before opening the Add Disks dialog, otherwise you might miss some pools from the list or have false volume list.

Reply:
Can you give me an example how to reproduce this? I tried and I still don't get the state which you are explaining.

Reply:
Let's say you are in the Disk Tab, from the CLI you remove a volume inside a pool, like rm pooldir/volumeA.qcow2 and then open the 'Add Disk' modal. You will still see that volume in the existing volumes, right?

Reply:
Tried what you described, and even with this pool-refresh call removed, after running "virsh vol-delete" and then opening Add Disk dialog, I get the corrent list of the volumes not containing the deleted one.

Reply:
I didn't say to run virsh vol-delete though :)
Maybe libvirt is smart enoigh (probably is) to refresh the pool after you use the vol-delete API.
Try doing that with rm volfile.

Reply:
That's true. Putting that refresh back